nx.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "./node_modules/nx/schemas/nx-schema.json",
+  "targetDefaults": {
+    "build": {
+      "dependsOn": [
+        "^build"
+      ],
+      "outputs": [
+        "{projectRoot}/dist"
+      ],
+      "cache": true
+    }
+  },
+  "parallel": 5,
+  "defaultBase": "main",
+  "plugins": [
+    {
+      "plugin": "@nx/rollup/plugin",
+      "options": {
+        "buildTargetName": "rollup:build"
+      }
+    }
+  ],
+  "nxCloudAccessToken": "Mzg3ZGY1MWUtYmYyNy00ZmE4LTkyNDAtYjYxZmJmYmE4NWQ3fHJlYWQ="
```

Comment:
Should this be committed? Is this a read only access token?

Reply:
Yeap this is a read only fallback token.

I had a chat with Tim, and we have a read/write access token in 1pass now, the nx-cloud.example.env now details how to get that one set up. We also have a read/write token set up for the github workflow set up in the secrets. Since our repo source code is public anyway the only thing this will allow people to access is the build output artifacts that they can get from manually building, and won't be able to push anything malicious to the cache server.

Reply:
Good to know, thanks @zaidarain1!